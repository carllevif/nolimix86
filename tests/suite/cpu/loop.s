  .text
mov $0, %eax
loop:
  cmpl $10, %eax
  je end
  add $1, %eax
  jmp loop
end:
