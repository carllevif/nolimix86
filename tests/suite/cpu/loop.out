---
cpu:             x86
registers:       
  - name:            ebp
    value:           0
  - name:            esi
    value:           0
  - name:            esp
    value:           0
  - name:            eax
    value:           10
  - name:            ebx
    value:           0
  - name:            ecx
    value:           0
  - name:            edi
    value:           0
flags:           
  - name:            pf
    value:           0
  - name:            af
    value:           0
  - name:            cf
    value:           0
  - name:            of
    value:           0
  - name:            sf
    value:           1
  - name:            zf
    value:           1
...
